index.html

```diff
@@ -41,9 +41,127 @@ <h2 class="green">I’m a software developer</h2>
         <div class="next-section">
             <img class="next-section-svg" src="assets/icons/round-arrow-down.svg" alt="">
         </div>
+        <div class="background-image-wrapper">
+            <img id="into-background" src="assets/images/background-mobile.svg" alt="">
```

Comment:
{Optional] I kindly suggest that we modify our intro section to match the Figma design we've chosen. Not only does it look better, but you'll also get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible. Therefore, sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now.

Our design currently:
![image](https://user-images.githubusercontent.com/20554583/186873019-13a08fea-c8f1-443a-9f27-f2404e44e847.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/186873186-b86192d7-0664-44da-b9ea-415f48d9edc1.png)

Reply:
I did the font size exactly as in figma.
I think it depends on device's width.

Reply:
Hi @udev-21,

The device with used to take the screenshot is same as the Figma design (375px)